packages/devtools_app/lib/src/timeline/flutter/timeline_processor.dart

```diff
@@ -0,0 +1,615 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
```

Comment:
update the diff to show the file this is based on assuming you are just forking a file that wasn't under the Flutter directory.

Reply:
See latest commit